packages/@coorpacademy-player-services/src/definitions.js

```diff
@@ -74,16 +74,29 @@ type StartProgressionEvent = {|
   }
 |};
 
-type FinishProgressionEvent = {|
-  event: 'finishProgression',
+type FinishedProgressionEvent = {|
+  event: 'finishedProgression',
   progression?: {
     type: string,
     state: string,
     extraLife: number
   }
 |};
 
-type DataEvent = MediaViewedEvent | StartProgressionEvent | FinishProgressionEvent;
+type UpdatedProgressionEvent = {|
+  event: 'updatedProgression',
```

Comment:
These updates must be made in mobile repository too

Reply:
you're right I put back the previous value